.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+about: Report a bug encountered using kind
+labels: kind/bug
+
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!-->
+
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
+
+**Environment:**
+
+- KIND version: (use `kind version`):
```

Comment:
maybe change `KIND` to `kind` ?

Reply:
Sure, I guess that we should use `Kind` to be consistent with the rest of the document: `Kubernetes` , `Docker`.  Is that ok?

Reply:
kind is usually stylized lowercase like the CLI. alternatively KIND but imho that's "shouty"

Reply:
kind then :)